core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3256,7 +3256,7 @@ private void setAclSingleInode(RpcContext rpcContext, SetAclAction action,
         targetInode.removeExtendedAcl();
         break;
       case REMOVE_DEFAULT:
-        targetInode.setDefaultACL(new DefaultAccessControlList());
```

Comment:
When is this empty constructor ever useful?

Reply:
it is useful when the client tries to deserialize the `DefaultAccessControlList` from the network